nva-datamodel-testutils/src/main/java/no/unit/nva/model/testing/PublicationGenerator.java

```diff
@@ -71,6 +77,18 @@ public static Publication randomPublication(Class<?> publicationInstanceClass) {
 
         return buildRandomPublicationFromInstance(publicationInstanceClass);
     }
+    public static Publication randomPublicationNonDegree() {
+        var degrees = Set.of(DegreeMaster.class,
+                             DegreeBachelor.class,
+                             DegreePhd.class,
+                             DegreeLicentiate.class,
+                             OtherStudentWork.class);
```

Comment:
In terms of accessRights ThesisCurators are the only ones supposed to edit an Degree - does this include "OtherStudentWork" - it inherits from BaseDegree